src/server/server.js

```diff
@@ -56,12 +57,12 @@ app.use(metricsBundle);
 
 // routes
 app.use('/metrics', trustedNetworks, metricsBundle.metricsMiddleware);
+app.use(generateToken);
 app.use('/', routes.login);
 app.use('/api', routes.github);
 app.use('/api', routes.launchpad);
 app.use('/api', routes.store);
 app.use('/', routes.webhook);
-app.use(routes.login);
 app.use(routes.githubAuth);
```

Comment:
It might also be worth considering where this goes in the route order, as with `routes.webhook` above.